### PR TITLE
docs(otel-binary): 📝 instructions for arm/amd in linux/macos

### PR DESCRIPTION
**Pull request by prashant-shahi**
Signed-off-by: Prashant Shahi <prashant@signoz.io>

### CODE REVIEW

File: docs/tutorial/opentelemetry-binary-usage.md [hunk 1]
```diff
@@ -38,124 +51,68 @@ OpenTelemetry-instrumented applications in a VM can send data to the `otel-binar
...
+4. <OtelExecuteMacos />
+
+5. <TailLogs />
+
+6. <OtelStop />
+
+</TabItem>
+</Tabs>
 
 Application Instrumentation → Otel-Binary Agent in Same VM → SigNoz Saas
```
Comment (ankit01-oss) [inline docs/tutorial/opentelemetry-binary-usage.md]:
```suggestion
Application Instrumentation → Otel-Binary Agent in Same VM → SigNoz Saas
```

Should we remove this line @prashant-shahi ?

Comment (prashant-shahi) [reply docs/tutorial/opentelemetry-binary-usage.md]: Actually a diagram would have been nice, and in the beginning of the section.

Comment (ankit01-oss) [reply docs/tutorial/opentelemetry-binary-usage.md]: Screenshot 2023-09-27 at 10 55 15 AM

This diagram is enough, right?

Comment (prashant-shahi) [reply docs/tutorial/opentelemetry-binary-usage.md]: yes, that looks nice. We can get rid of it.

Later we should add similar diagram for SigNoz OSS as well.

**Review by ankit01-oss — APPROVED** [review]

**Pull request merged**